percona/controllers/replica/replica.go

```diff
@@ -249,7 +315,7 @@ func getReplicaServiceName(clusterName string) string {
 	return fmt.Sprintf("%s-replica", clusterName)
 }
 
-func UpdateResources(cl *crv1.PerconaPGCluster, deployment *appsv1.Deployment) error {
+func updateResources(cl *crv1.PerconaPGCluster, deployment *appsv1.Deployment) error {
```

Comment:
you always return nil. Is it's really needed to indicate potential error?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/71/commits/a8cbb68cc0989a1d752f02faecd671470266675a